src/core/issues-notes.js

```diff
@@ -203,6 +193,36 @@ function createIssueSummaryEntry(l10nIssue, report, id) {
   `;
 }
 
+/**
+ *
+ * @param {*} issueList
+ */
+function makeIssueSectionSummary(issueList) {
+  const issueSummaryElement = document.getElementById("issue-summary");
+
+  if (issueSummaryElement) {
+    if (document.querySelectorAll(".issue").length) {
+      const heading = issueSummaryElement.querySelector("h2");
+      const paragraph = issueSummaryElement.querySelector("p");
```

Comment:
Likewise, there is no restriction on what content there can be inside the summary:

```HTML
<section>
   <h3>Appendix</h3>
   <section id="issue-summary">
     <h4>OMG so many issues!</h4>
     <aside class="issue">Do we have too many issues?</aside>
     <figure><figcaption>A diagram</figcaption></figure>
   </section>
</section> 
```